subprojects/language-native/src/integTest/groovy/org/gradle/language/swift/SwiftIncrementalBuildIntegrationTest.groovy

```diff
@@ -36,10 +36,12 @@ import org.gradle.test.fixtures.file.TestFile
 import org.gradle.util.Requires
 import org.gradle.util.TestPrecondition
 import org.gradle.util.internal.VersionNumber
+import spock.lang.Ignore
 
 @RequiresInstalledToolChain(ToolChainRequirement.SWIFTC)
 // https://github.com/gradle/gradle-private/issues/3387
 @Requires(TestPrecondition.NOT_EC2_AGENT)
+@Ignore // TODO (donat) re-enable once CI gets fixed
```

Comment:
I needed to disable this test as it has problems on CI. I'll remove this change once the issue is solved.